tools/scripts/bumpenvs.yaml

```diff
@@ -142,6 +142,18 @@ tf2_gpu_hashed:
 tf2_gpu_versioned:
   new: determinedai/environments:cuda-11.1-pytorch-1.9-lightning-1.5-tf-2.4-gpu-0.17.10
   old: determinedai/environments:cuda-11.1-pytorch-1.9-lightning-1.3-tf-2.4-gpu-0.17.9
+deepspeed_gpu_hashed:
+  new: determinedai/environments:cuda-11.1-pytorch-1.9-lightning-1.5-tf-2.4-gpu-5158dec
+  old: determinedai/environments:cuda-11.1-pytorch-1.9-lightning-1.5-tf-2.4-gpu-9f0cb26
+deepspeed_gpu_versioned:
+  new: determinedai/environments:cuda-11.1-pytorch-1.9-lightning-1.5-tf-2.4-gpu-0.17.10
+  old: determinedai/environments:cuda-11.1-pytorch-1.9-lightning-1.5-tf-2.4-gpu-0.17.9
+deepspeed_gpt_neox_gpu_hashed:
```

Comment:
Why do we have this and also a `gpt_neox_deepspeed_gpu_hashed` entry up higher?

Reply:
my mistake.  looks like i edited the file twice and forgot i already started to add deepspeed images the second time :disappointed: